src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
I agree that `html5-history` is a better name. I didn't see `html5` prefixes elsewhere, so I left it off. But I certainly wouldn't mind adding it.